scripts/hermit_test.py

```diff
@@ -1,86 +1,150 @@
 #!/usr/bin/env python
-import os
+import argparse
+import re
 import sys
-from optparse import OptionParser
+from pathlib import Path
+
+
+OWL_NOTHING = 'http://www.w3.org/2002/07/owl#Nothing'
+BOTTOM_OBJECT_PROPERTY = 'http://www.w3.org/2002/07/owl#bottomObjectProperty'
+
+IRI = '<(.*?)>'
+
+EQUIVALENT_CLASSES = r'^EquivalentClasses\( (.*) \)$'
+OBJECT_INVERSE_OF = r'ObjectInverseOf\( ' + IRI + r' \)'
+EQUIVALENT_OBJECT_PROPERTIES = r'^EquivalentObjectProperties\( (.*) \)$'
+
+
+def equivalent_to_nothing(subsumption):
+    """Searches for all classes which are equivalent to owl#Nothing"""
+    results = re.search(EQUIVALENT_CLASSES, subsumption).group(1)
+    classes = set(re.findall(IRI, results))
+    return OWL_NOTHING in classes, classes
+
+
+def subclass_of_nothing(subsumption):
+    """Searches for all classes which are subclasses of owl#Nothing"""
+    return OWL_NOTHING in subsumption, set(re.findall(IRI, subsumption)[0])
+
+
+def equivalent_to_bottom_object_property(subsumption):
```

Comment:
similar to the case above, bottomObjectProperty should be excluded in the list of properties that are equivalent to bottomObjectProperty

Reply:
same as above: hermit does not include that.